src/utils/obj_ctl.c

```diff
@@ -553,6 +554,6 @@ main(int argc, char **argv)
 	return rc;
 
  out_usage:
-	printf("%s daos|vos\n", argv[0]);
-	return -1;
+	fprintf(stderr, "%s daos|vos\n", argv[0]);
```

Comment:
is this fprintf call redundant with the one above, line 507, if (argc < 3)? Also that printf has 2 string placeholders: %s %s daos|vos (filled with argv[0] and argv[1]) but this call has just one %s placeholder filled with argv[0].

Reply:
Done